src/components/Button/Button.js

```diff
@@ -33,31 +35,31 @@ const ButtonImpl = styled.button`
   overflow: hidden;
 
   &:hover:not(:disabled), &:focus {
-    background-color: ${({ theme }) => theme.button.activeBG};
-    border-color: ${({ theme }) => theme.button.activeBG};
-    color: ${({ theme }) => theme.button.activeFG};
-    box-shadow: 0 2px 4px ${({ theme }) => theme.shadow.color};
+    background-color: ${({ theme }) => theme.colors.primaryDark};
+    border-color: ${({ theme }) => theme.colors.primaryDark};
+    color: ${({ theme }) => theme.colors.white};
+    box-shadow: 0 2px 4px rgba(0, 0, 0, 0.14);
```

Comment:
A nearly identical drop shadow of `rgba(0,0,0,0.14)` appears in the baseTheme under `shadows.default`. Should we extract the color into its own value?